scripts/develop.sh

```diff
@@ -36,33 +42,33 @@ fi
 (
 	# If something goes wrong, just bail and tear everything down
 	# rather than leaving things in an inconsistent state.
-	trap 'kill -INT -$$' ERR
+	trap 'kill -TERM -$$' ERR
```

Comment:
I think this will prevent graceful shutdown of the server (e.g. resource cleanup), do we want to avoid that?

Reply:
Should only trigger on an error and if we encounter an error then stuff hasn't even managed to start, so we should be fine.

Reply:
Oh right, that makes sense! Ignore me. 😄